userspace/libscap/engine/gvisor/CMakeLists.txt

```diff
@@ -60,10 +60,10 @@ add_library(
 	scap_engine_gvisor ${scap_engine_gvisor_sources} ${scap_engine_gvisor_generated_sources}
 )
 
-add_dependencies(scap_engine_gvisor uthash jsoncpp)
+add_dependencies(scap_engine_gvisor uthash jsoncpp scap_event_schema)
```

Comment:
in an initial attempt, i tried to add gvisor to scap open but I faced a link error for `scap_event_encode_params` so it seems correct to link against `scap_event_schema`